astrobase/varbase/trends.py

```diff
@@ -101,16 +102,27 @@ def LOGEXCEPTION(message):
 ## SMOOTHING FUNCTIONS ##
 #########################
 
-def smooth_magseries_medfilt(mags, windowsize):
+def smooth_magseries_median_filter(mags, windowsize):
```

Comment:
Maybe rename this to `smooth_magseries_ndimage_medfilt` to make this more explicit. I'll rename the other one to `smooth_magseries_signal_medfilt`.

Reply:
done